resources/todomvc/architecture-examples/svelte/package.json

```diff
@@ -11,7 +11,7 @@
         "build": "rollup -c",
         "watch": "rollup -c -w",
         "serve": "http-server ./dist -p 7002 -c-1 --cors",
-        "dev": "npm run serve & npm run watch"
+        "dev": "npm run serve & npm run watch --no-watch.clearScreen"
```

Comment:
@julienw - let me know if this works for you

Reply:
it threw an error for me 🤷

Reply:
very weird because this worked for me:
```diff
diff --git a/resources/todomvc/architecture-examples/svelte/rollup.config.js b/resources/todomvc/architecture-examples/>
index b9bf6c67..4c1f9c26 100644
--- a/resources/todomvc/architecture-examples/svelte/rollup.config.js
+++ b/resources/todomvc/architecture-examples/svelte/rollup.config.js
@@ -12,16 +12,19 @@ const production = !process.env.ROLLUP_WATCH;
 export default {
     input: "src/index.js",
     output: {
         file: "dist/app.js",
         format: "iife",
         sourcemap: true,
         name: "app",
     },
+    watch: {
+        clearScreen: false,
+    },
     plugins: [
         css({
             minify: true,
         }),
         svelte({
             include: "src/**/*.svelte",
         }),
         resolve({
```

Reply:
maybe it took a while for the coffee to kick in, but I moved it in the config and it works of course.